xslt/Channels.xsl

```diff
@@ -72,6 +72,13 @@
         <xsl:param name="idx"/>
         <xsl:element name="ome:Channel">
             <xsl:attribute name="ID">
+                <!-- Prepend 'Channel:' if it isn't contained within the ID in order to satisfy OME spec -->
+                <xsl:variable name="channel_id">
+                    <xsl:value-of select="@Id"/>
+                </xsl:variable>
+                <xsl:if test="not(contains($channel_id,'Channel:'))">
+                    <xsl:text>Channel:</xsl:text>
```

Comment:
How does this interact, if at all, with https://github.com/AllenCellModeling/aicsimageio/blob/main/aicsimageio/metadata/utils.py#L150?

Reply:
Hm, I'm not entirely sure and might need some input from @JacksonMaxfield. 

I'll continue to investigate, but I'm not super familiar with `aicsimageio`

Reply:
It's only used for writing or "fake" image construction:
![image](https://user-images.githubusercontent.com/17132317/158709688-aeddb8f7-609d-433e-a412-b78943a7518a.png)

"fake" image or metadata construction happens for all the "non-metadata" readers, `ArrayLike`, `Default`, `TiffGlob` and `Tiff` and in the `AICSImage` object the underyling image data doesn't have a channel dimension and we are inserting an empty channel dim.

only other time its used is during image writing to OME-TIFF (and OME-Zarr).

Reply:
The change in this PR on the other hand, would only affect the returned value of `ome_metadata` from the `CziReader` object (and parent `AICSImage` object)